webpack/ForemanInventoryUpload/Components/PageHeader/components/CloudConnectorButton/CloudConnectorSelectors.js

```diff
@@ -1,14 +1,20 @@
 import { selectAPIResponse } from 'foremanReact/redux/API/APISelectors';
+import { foremanUrl } from '../../../../../ForemanRhCloudHelpers';
 
 import {
   CONFIGURE_CLOUD_CONNECTOR,
   CONNECTOR_STATUS,
 } from './CloudConnectorConstants';
 
-export const selectStatus = state =>
-  selectAPIResponse(state, CONFIGURE_CLOUD_CONNECTOR)?.result === 'success'
-    ? CONNECTOR_STATUS.RESOLVED
-    : CONNECTOR_STATUS.NOT_RESOLVED;
+export const selectStatus = state => {
+  const { task } = selectAPIResponse(state, CONFIGURE_CLOUD_CONNECTOR);
```

Comment:
We will need to add this data into the polling that happens on the page,
otherwise, after the job succeeds/fails the button won't update until the next page refresh.

Reply:
For now thats what we want, this is the next step

Reply:
it's a bit misleading, now that we add the spinner and it can load for ages... while the job was actually successful after 1-2 minutes

Reply:
@laviro You're right about this. 
I think we should drop the spinner then, and change the message to something more neutral: `Cloud connector setup started` fir example.

Reply:
it's still a bit confusing I guess,
At this stage, I would suggest adding it to the polling group in [accounts controller](https://github.com/theforeman/foreman_rh_cloud/blob/master/app/controllers/foreman_inventory_upload/accounts_controller.rb#L29) and [accounts reducer](https://github.com/theforeman/foreman_rh_cloud/blob/3a48a02c147d33aa553f2da7e28d8358e22ac3b9/webpack/ForemanInventoryUpload/Components/AccountList/AccountListReducer.js#L36).
And in the next iteration, we should refactor it out of accounts.

Reply:
That's an option... @MariaAga, if it sounds doable for you, let's go with this solution